Dockerfile

```diff
@@ -2,10 +2,10 @@ ARG BASE_IMAGE_FULL
 ARG BASE_IMAGE_MINIMAL
 
 # Build node feature discovery
-FROM golang:1.17.13-buster as builder
+FROM golang:1.17.13 as builder
```

Comment:
Why not upgrade to golang 1.19 or 1.20 if it works since these versions are still getting security updates (it's okay to stay at 1.17 if it doesn't work)

Reply:
You can upgrade the golang upgrade in that case - I don't want to delay the PR to just catch up with the latest if there are no HIGH/ CRIT vulns due to golang right now.

Reply:
have upgraded to golang:v1.19